Source/Plugins/ImportExport/ImportExportFilters/NodesTrianglesToVtk.cpp

```diff
@@ -109,50 +109,44 @@ void NodesTrianglesToVtk::initialize()
 // -----------------------------------------------------------------------------
 void NodesTrianglesToVtk::dataCheck()
 {
-  setErrorCondition(0);
-  setWarningCondition(0);
+  clearErrorCode();
+  clearWarningCode();
 
   QFileInfo fi(m_TrianglesFile);
   if(m_TrianglesFile.isEmpty())
   {
-    setErrorCondition(-1000);
-    notifyErrorMessage(getHumanLabel(), "Triangles file is not set correctly", -1001);
+    setErrorCondition(-1001, "Triangles file is not set correctly");
   }
   else if(!fi.exists())
   {
 
     if(getInPreflight())
     {
-      setWarningCondition(-1001);
       QString ss = "Triangles file does not exist currently.\nYou must have another filter that creates these files before this filter in your pipeline";
-      notifyWarningMessage(getHumanLabel(), ss, getWarningCondition());
+      setWarningCondition(-1002, ss);
     }
     else
     {
-      setErrorCondition(-1002);
-      notifyErrorMessage(getHumanLabel(), "Triangles file does not exist currently.\nYou must have another filter that creates these files before this filter in your pipeline", -1004);
+      setErrorCondition(-1003, "Triangles file does not exist currently.\nYou must have another filter that creates these files before this filter in your pipeline");
     }
   }
 
   QFileInfo fii(m_NodesFile);
   if(m_NodesFile.isEmpty())
   {
-    setErrorCondition(-1003);
-    notifyErrorMessage(getHumanLabel(), "Nodes file path or name is emtpy", -1002);
+    setErrorCondition(-1004, "Nodes file path or name is emtpy");
   }
   else if(!fii.exists())
   {
 
     if(getInPreflight())
     {
-      setWarningCondition(-1004);
       QString ss = "Nodes file does not exist currently. You must have another filter that creates these files before this filter in your pipeline";
-      notifyWarningMessage(getHumanLabel(), ss, getWarningCondition());
+      setWarningCondition(-1005, ss);
```

Comment:
Why did this error code change?

Reply:
I changed all the error codes in that file because they either weren't set at all before emitting the error message or they were set to the exact same error code as a previous error message (which isn't helpful at all).  I changed them so that they are all unique error codes, and then I ran the unit tests to make sure it doesn't break any of those.

Reply:
Aren't warning conditions > 0? I can't remember.

Reply:
We haven't been using that distinction.  I think the idea that all errors are negative bled over into warnings as well.